packages/suite/jest.config.js

```diff
@@ -31,9 +33,7 @@ module.exports = {
         '^src/(.+)': '<rootDir>/src/$1',
         '\\.(mp4)$': '<rootDir>/__mocks__/import-mp4.js',
         '\\.(svg)$': '<rootDir>/__mocks__/import-svg.js',
-        uuid: require.resolve('uuid'), // https://stackoverflow.com/questions/73203367/jest-syntaxerror-unexpected-token-export-with-uuid-library
-        // Enforce usage of JS version of bcrypto in tests because on CI we don't build native modules because it's slowing yarn install
-        '^bcrypto/lib/(.*)$': 'bcrypto/lib/$1-browser',
```

Comment:
This repeats what is in `baseConfig`, and now I needed to also add the `uint8array-tools` everywhere. So let's keep it DRY.